spec/formatting.md

```diff
@@ -92,18 +92,20 @@ and different implementations MAY choose to perform different levels of resoluti
 > or some other locally appropriate value.
 
 Depending on the presence or absence of an _operand_
-and a _function_ or _reserved_ _annotation_,
+and a _function_, _private-use_, or _reserved_ _annotation_,
 one of the following is used to resolve the value of the _expression_:
 
 - If the _expression_ contains no _annotation_,
   its resolved value is determined by _literal resolution_ or _variable resolution_,
   depending on the shape of the _operand_.
 - Else, if the _expression_ has a _function_ _annotation_,
   its resolved value is defined by _function resolution_.
-- Else, the _expression_ has a _reserved_ _annotation_.
-  If the _annotation_ uses a `private-start` character that the implementation supports,
-  its value is resolved according to the implementation's specification.
-  Else, an Unsupported Expression error is emitted and a fallback value is used as its value.
+- Else, if the _expression_ has a _private-use_ _annotation_,
+  its resolved value is defined according to the implementations's specification.
+- Else, if the _expression_ has a _reserved_ _annotation_,
+  an Unsupported Expression error is emitted and a fallback value is used as its value.
+  Else the contents are syntactically invalid and
+  an Unsupported Expression error is emitted and a fallback value is used as its value.
```

Comment:
I don't think "syntactically invalid" can apply here. If we'd encountered a syntax error, then we wouldn't have an _expression_ to resolve. We could in theory still get here if the message had been passed in as a (broken) data model and the implementation didn't sufficiently validate such, but then we can't presume that it originally came from any syntax and should treat it as a data model error.

In either case, we can trust that pattern selection has already resolved such a broken message to a fallback value.

```suggestion
- Else, the _expression_ has a _reserved_ _annotation_.
  An Unsupported Expression error is emitted and a fallback value is used as its value.
```

Reply:
Done.